modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneCfg.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.distributionzones;
+
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Distribution zone configuration.
+ */
+public class DistributionZoneCfg {
+    /** Zone name. */
+    private final String name;
+
+    /** Data nodes auto adjust timeout. */
+    private final Integer dataNodesAutoAdjust;
+
+    /** Data nodes auto adjust scale up timeout. */
+    private final Integer dataNodesAutoAdjustScaleUp;
+
+    /** Data nodes auto adjust scale down timeout. */
+    private final Integer dataNodesAutoAdjustScaleDown;
+
+    /**
+     * The constructor.
+     */
+    public DistributionZoneCfg(String name,
+                               Integer dataNodesAutoAdjust,
+                               Integer dataNodesAutoAdjustScaleUp,
+                               Integer dataNodesAutoAdjustScaleDown) {
+        this.name = name;
+        this.dataNodesAutoAdjust = dataNodesAutoAdjust;
+        this.dataNodesAutoAdjustScaleUp = dataNodesAutoAdjustScaleUp;
+        this.dataNodesAutoAdjustScaleDown = dataNodesAutoAdjustScaleDown;
+    }
+
+    /**
+     * Gets the zone name.
+     *
+     * @return The zone name.
+     */
+    @Nullable
```

Comment:
This should on the same line with method

Reply:
fixed